docs/index-en.html

```diff
@@ -0,0 +1,4245 @@
+          <dt>Issued on: </dt>
+          <dd>2022-04-03</dd>
+          <dt>Authors:</dt>
+          <dd>Tiago Prince Sales, (<a href="https://www.utwente.nl/en/eemcs/scs/">Semantics,
+              Cybersecurity &amp; Services (SCS), University of Twente, the
+              Netherlands</a>)</dd>
+          <dd>Claudenir M. Fonseca, (<a href="https://www.utwente.nl/en/eemcs/scs/">Semantics,
+              Cybersecurity &amp; Services (SCS), University of Twente, the
+              Netherlands</a>)</dd>
+          <dd>Pedro Paulo Favato Barcelos, (<a href="https://www.utwente.nl/en/eemcs/scs/">Semantics,
+              Cybersecurity &amp; Services (SCS), University of Twente, the
+              Netherlands</a>)</dd>
+          <dt>Contributors:</dt>
+          <dd>Mattia Fumagalli, (<a href="https://www.inf.unibz.it/krdb/">KRDB
+              Research Centre for Knowledge and Data, Free University of
+              Bozen-Bolzano, Italy</a>)</dd>
+          <dt>Publisher:</dt>
+          <dd><a href="https://www.utwente.nl/en/eemcs/scs/">Semantics,
+              Cybersecurity &amp; Services (SCS), University of Twente, the
+              Netherlands</a></dd>
+          <dt>Download serialization:</dt>
+          <dd><span><a href="ontology.jsonld" target="_blank"><img src="https://img.shields.io/badge/Format-JSON_LD-blue.svg"
+
+                  alt="JSON-LD"></a> </span><span><a href="ontology.owl" target="_blank"><img
+
+                  src="https://img.shields.io/badge/Format-RDF/XML-blue.svg" alt="RDF/XML"></a>
+            </span><span><a href="ontology.nt" target="_blank"><img src="https://img.shields.io/badge/Format-N_Triples-blue.svg"
+
+                  alt="N-Triples"></a> </span><span><a href="ontology.ttl" target="_blank"><img
+
+                  src="https://img.shields.io/badge/Format-TTL-blue.svg" alt="TTL"></a>
+            </span></dd>
+          <dt>License:</dt>
+          <dd><a href="https://creativecommons.org/licenses/by-sa/4.0" target="_blank"><img
+
+                src="https://img.shields.io/badge/License-https://creativecommons.org/licenses/by_sa/4.0-blue.svg"
+
+                alt="https://creativecommons.org/licenses/by-sa/4.0"></a> </dd>
+          <!-- <dt>Evaluation:</dt><dd><a href="OOPSevaluation/oopsEval.html#" target="_blank"><img src="https://img.shields.io/badge/Evaluate_with-OOPS! (OntOlogy Pitfall Scanner!)-blue.svg" alt="Evaluate with OOPS!" /></a></dd> --><dt>Cite
+            as:</dt>
+          <dd>Sales, T. P., Fonseca, C. M., Barcelos, P. P. F. (2022) OntoUML Vocabulary Specification - Version 1.1.0. <a href="https://w3id.org/ontouml/1.1.0">https://w3id.org/ontouml/1.1.0</dd>
+        </dl>
+        <a href="provenance/provenance-en.html" target="_blank">Provenance of
+          this page</a>
+        <hr> </div>
+      <div class="status">
+        <div> <span>OntoUML Vocabulary Specification - Version 1.1.0</span> </div>
```

Comment:
I think we should remove this floating title on the top-left corner of the page.

Reply:
This is part of the W3C template. Its recommendations use this floating title. IMO, we should keep to be aligned with them.

However, @tgoprince and @claudenirmf, if you two decide to remove, I won't oppose to that.

Reply:
Why would we want to stick we the W3C template? We are not associated with them.

Reply:
Done